connector-j-5/pom.xml

```diff
@@ -19,6 +19,26 @@
     SSL certificates manually.
   </description>
 
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.guava</groupId>
+        <artifactId>guava</artifactId>
+        <version>29.0-android</version>
```

Comment:
Are we sure that android is the version we should be using here?

Reply:
That was the version that came up when running maven-enforcer-plugin and I just copied it. I could just change it to version 29.0.

Reply:
android might be the right call, I just want to be sure. What does the sql-admin-api use? We should likely match that.